app/controllers/application_controller/ci_processing.rb

```diff
@@ -1804,6 +1824,12 @@ def scanhosts
     host_button_operation('scan', _('Analysis'))
   end
 
+  # Toggle maintenance mode on all selected or single displayed host(s)
+  def maintenancehosts
+    assert_privileges("host_edit")
```

Comment:
Shouldn't we be checking `"host_toggle_maintenance"` rather than "host_edit" here?